projects/storefrontstyles/scss/components/myaccount/order-history/_replenishment-order-history.scss

```diff
@@ -0,0 +1,200 @@
+%cx-replenishment-order-history {
+  .cx-replenishment-order-history {
+    @include media-breakpoint-down(sm) {
+      max-width: 100%;
+      padding: 0;
+    }
+  }
+  .cx-replenishment-order-history-table {
+    padding: 1.5rem 0 1.125rem 0;
+    margin-bottom: 0;
+    border-width: 1px 0 1px 0;
+    border-color: var(--cx-color-light);
+    border-style: solid;
+
+    tr {
+      width: 100%;
+
+      @include media-breakpoint-down(sm) {
+        border-width: 1px 0 0 0;
+        border-color: var(--cx-color-light);
+        border-style: solid;
+
+        &:first-child {
+          border-width: 0;
+          padding: 1.25rem 0 0 0;
+        }
+      }
+    }
+
+    th {
+      padding: 1.5rem 0 1.125rem 0;
+      padding-top: 1.5rem;
+      padding-inline-end: 0;
+      padding-bottom: 1.125rem;
+      padding-inline-start: 5px;
+      text-align: left;
+
+      &:first-child {
+        padding-inline-start: 0;
+      }
+
+      &:last-child {
+        text-align: right;
+      }
+    }
+
+    td {
+      width: 14.2%;
+      padding: 1.625rem 5px;
+
+      @include media-breakpoint-up(md) {
+        text-align: left;
+        &:first-child {
+          padding-inline-start: 0px;
+        }
+
+        &:last-child {
+          text-align: right;
+        }
+      }
+
+      @include media-breakpoint-down(sm) {
+        width: 100%;
+        display: flex;
+        border-width: 0;
+        padding: 0 1.25rem;
+
+        &:first-child {
+          padding-top: 1.25rem;
+        }
+
+        &:last-child {
+          padding-bottom: 1.25rem;
+          padding-top: 7px;
+        }
+      }
+    }
+  }
+
+  .cx-replenishment-order-history-header {
+    padding: 40px 0 0 0;
+    color: var(--cx-color-text);
+
+    @include media-breakpoint-down(sm) {
+      padding: 40px 20px 0 20px;
+    }
+  }
+
+  .cx-replenishment-order-history-thead-mobile {
+    @include media-breakpoint-down(sm) {
+      display: none;
+    }
+  }
+
+  .cx-replenishment-order-history-code {
+    .cx-replenishment-order-history-value {
+      text-decoration: underline;
+    }
+  }
+
+  .cx-replenishment-order-history-placed {
+    text-align: center;
+
+    @include media-breakpoint-down(sm) {
+      text-align: left;
+    }
+  }
+
+  .cx-replenishment-order-history-status {
+    text-align: center;
+
+    @include media-breakpoint-down(sm) {
+      text-align: left;
+    }
+  }
+
+  .cx-replenishment-order-history-total {
+    text-align: right;
+
+    @include media-breakpoint-down(sm) {
+      text-align: left;
+    }
+  }
+
+  .cx-replenishment-order-history-label {
+    text-transform: uppercase;
+    color: var(--cx-color-secondary);
+
+    @include media-breakpoint-down(sm) {
+      @include type('8');
+      min-width: 60%;
```

Comment:
if you make the window screen smaller to make it pop to mobile view (without clicking the mobile tool in dev tools), you can see it looks pretty weird with a lot of spacing compared to its counterpart 

![Screen Shot 2020-09-23 at 3 39 56 PM](https://user-images.githubusercontent.com/29099466/94066266-3d280a00-fdba-11ea-9c83-6c5905fe3b4c.png)

![Screen Shot 2020-09-23 at 4 31 38 PM](https://user-images.githubusercontent.com/29099466/94066290-4618db80-fdba-11ea-9e61-f3e0c3084283.png)

Reply:
set it to 50%. anything smaller it making the values run into lables on mobile